drivers/netdev_ieee802154/netdev_ieee802154.c

```diff
@@ -50,6 +51,19 @@ static int _get_iid(netdev_ieee802154_t *dev, eui64_t *value, size_t max_len)
     return sizeof(eui64_t);
 }
 
+void netdev_ieee802154_reset(netdev_ieee802154_t *dev)
+{
+    dev->seq = random_uint32();
```

Comment:
Spend a comment that only one (the lowest) byte of the random value is used

Reply:
Done! Please let me know if it is sufficiently clear.

Reply:
Fine with me. It's self-explaning anyway, but for clarity reasons...